pkg/apis/admissionregistration/types.go

```diff
@@ -196,7 +196,7 @@ type Webhook struct {
 	// on whether the namespace for that object matches the selector. If the
 	// object itself is a namespace, the matching is performed on
 	// object.metadata.labels. If the object is other cluster scoped resource,
-	// it is not subjected to the webhook.
+	// it never skips the webhook.
```

Comment:
@sttts PTAL. Sorry it has been off my radar for a while.

Reply:
nit: another cluster scoped resource.

But I am no native speaker either @deads2k you are. Can you comment?

Reply:
Confirmed with @cheftako that we should use "another" :)

Reply:
:)